docs/api-spec-generated/security.md

```diff
@@ -23,5 +23,41 @@ scopes in the future such as limiting access to particular event types and strea
 |nakadi.event_stream.write|Grants access for applications to submit Events.|
 |nakadi.event_stream.read|Grants access for consuming Event streams.|
 
-
-
+<a name="event type"></a>
+### Event Type
+#### Write and read scopes
+
+Nakadi has possibility to check OAuth2 Scopes (scopes) of publisher and consumer to give access
+write and read events only for trusted users.
+In order to use this feature an event type owner is able to specify read and write
+scopes for the event type:
+
+```sh
+curl -v -XPOST http://localhost:8080/event-types -d '{
+  "name": "order_received",
+  "owning_application": "acme-order-service",
+  "category": "business",
+  "partition_strategy": "random",
+  "enrichment_strategies": ["metadata_enrichment"],
+  "schema": {
+    "type": "json_schema",
+    "schema": "{ \"properties\": { \"order_number\": { \"type\": \"string\" } } }"
+  }
+
+  "write_scopes" : ["oauth2.write.scope.one", "oauth2.write.scope.two"],
+  "read_scopes" : ["oauth2.read.scope.one", "oauth2.read.scope.two"]
+}'
+```
+
+Once publisher tries to write event to `order_received` event type he is examined for scopes
```

Comment:
What about "...tries to write *an* event to `order_receive` *,* its oath token is examined for scopes..."?